### DISCUSSION TITLE
Meta information about photos

### DISCUSSION THREAD

**Discussion by FreeWind6** [alangrainger/immich-public-proxy#27] [Feature Requests]
Please add meta information about photos/videos

**Comment by Hidden-Warden** [comment]
It could be nice indeed to be able to see the description and location of the media, like it seems to be possible with lightGallery.

**Comment by Danie10** [comment]
Yes I would love this as well

**Comment by alangrainger** [comment]
Photo description has been added in 1.11.0

https://github.com/alangrainger/immich-public-proxy/releases/tag/v1.11.0

**Reply by Danie10** [reply]
It's working perfectly — what a game changer this is now for Immich!! Just used the updated config.json provided, and it worked as expected.

![image](https://github.com/user-attachments/assets/c4f85e49-257f-4b32-bfff-c2941384a0eb)

**Reply by ovizii** [reply]
> It's working perfectly — what a game changer this is now for Immich!! Just used the updated config.json provided, and it worked as expected.

Would you mind sharing your config.json? The docs are very confusing. Not sure what to make of this:

<img width="694" height="47" alt="image" src="https://github.com/user-attachments/assets/b2b0b0df-67a8-4172-b894-84b91cfc8ed9" />

coupled with this:

<img width="749" height="154" alt="image" src="https://github.com/user-attachments/assets/eae04e54-2a14-448e-8d87-45cfabf532df" />

How is this supposed to look like?

   
I currently tested these two:
```
    "showGalleryTitle": true,
    "description": true,

```

Not sure what to do about: `    "showMetadata": ?`

**Reply by Danie10** [reply]
@ovizii I have links to the config as well as Docker compose that I'm using in the details on my video about it at https://www.youtube.com/watch?v=VkPtL4lxszA. I don't think there have been major changes yet - I'm still waiting hopefully to see the album description get pulled through.

**Reply by ovizii** [reply]
Cool, thanks. Works so far but is the description the only metadata we can currently show?

**Reply by Danie10** [reply]
@ovizii yes I have the latest version and there is no album description or anything else showing yet.

**Reply by ovizii** [reply]
Well, that seemed obvious seeing that I got my config from you, and we only define description, hence my question if other metadata could also be referenced. OR maybe I am missing something here? Thanks for helping me get this far though!

```
    "showMetadata": {
      "description": true
    }

```

**Reply by Danie10** [reply]
I think @alangrainger can hopefully give an update as I've not seen anything going live in the change logs yet. There can be options available in the config, but they may rely on back end functionality going live. The change he referred to above was for the photo descriptions/captions, not album descriptions.

So right now metadata info is the photo caption but not location, date, or other metadata. The original request issue did not specify anything specific, and I'm not sure if there are limitations imposed on the Immich side.

**Comment by Danie10** [comment]
@alangrainger one last thing on metadata, would it be possible to also pull through the album description as well?

Also if you could maybe test it with 100+ photos. I have an album of 174 photos, and I noticed it stops showing the caption after I've looked through a whole lot of photos (not at any specific point). It will still show captions for ones it showed before, but none of the ones where the caption does not show, ten to show up after reviewing. In other words, even if I refresh the album page, the ones that did not show their captions, still don't show. It seems more related to the number of photos I have opened, and not the limit by sequential order of how the photos are displayed. I also tried using just the IP address instead of https URL but seems the same.

**Comment by Danie10** [comment]
Only odd thing for metadata then is for the large albums where it seems to load well up to 35 or 40 captions, but then no longer shows it.  I tried a new share for about 104 photos with metadata enabled for share (at https://photoshare.gadgeteerza.co.za/share/le-rldJqTemYd68bwq-CjboGcz5INciYf-l74e5JSu6APmNArsd31iMwmhUR9lWuCp4) but now seems no caption is showing again. Can see it is definitely IPP displaying. Shows about the first 50 captions.

Smaller album at https://photoshare.gadgeteerza.co.za/share/I0WZJf9W98k_C_dBZzMhyMFjgDow5dsTYOZjpjNikhU5m0ho_ojVO7kXEgQSWYE_-vw?at=e2d62d61-d1bd-41c2-afb2-088bb87964a7 is showing captions OK.

**Reply by alangrainger** [reply]
This should be fixed in 1.11.1

**Comment by Danie10** [comment]
OK thanks tested. Yes it seems to work but not constantly:

1. On a private album with 174 photos, with a share link, that one is working 100%. I can jump back and forth and go to the end, caption is working perfectly across everything.
2. With the 104 photo sat public album at https://photoshare.gadgeteerza.co.za/share/le-rldJqTemYd68bwq-CjboGcz5INciYf-l74e5JSu6APmNArsd31iMwmhUR9lWuCp4 - it keeps stopping exactly at photo 50. If I open the photo and go to next it shows "Oops failed to load content". In the container logs I see "Invalid share key undefined". I tried sharing that same album with a new link, but got the same result. I don't see anything odd about photo number 51. I edited its caption and refreshed the shared view, but it still stops at 50.
3. A brand new share at https://photoshare.gadgeteerza.co.za/share/L0F7Pz6CF1ayhhpMPxyeRrXU6g68G68E9v3ptWCE2865xyBl1LraXPLRikZd-lFY6gg is doing the same. Only displays up to photo 50 and then gives same message and invalid key error in container log. I did also restart the whole stack, and put a depend line in to make sure IPP waits for Immich Server to be running.

**Reply by Danie10** [reply]
Looks like the above is all sorted out now after 2 days. They are viewing OK. Really not too sure why as the software is the same. I'm thinking browser caching maybe, but I did try two browsers, and it was showing an invalid key error. But both albums above are showing all photos and captions up to tech last one.

Just hoping for metadata that there is a way to maybe also bring the album description along with the album name (although it shows the album share name I think actually, which is fine).

**Comment by Ratterbass** [comment]
Are there any plans to pull the remaining metadata as well? It would be nice to show location, exif data etc. As far as I can tell this is currently not possible.

**Comment by alangrainger** [comment]
Hi all - yes, this is planned to add additional EXIF fields this month, sorry it's taken so long!

**Comment by alangrainger** [comment]
This is now live in 2.3.0 :)

https://github.com/alangrainger/immich-public-proxy/releases/tag/v2.3.0
